feat(Curriculum) Add A1 Spanish Module 1 (#62201)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/blocks/es-a1-practice-greetings-and-farewells/68c9881715d314368f261622.md

---
id: 68c9881715d314368f261622
title: Task 2
challengeType: 19
dashedName: task-2
---
<!-- (Audio) Camila ES_A1_1.2 -->

# --instructions--

Listen to the audio and answer the question below.

# --questions--

## --text--

What time of day is Camila speaking?

## --answers--

Afternoon

---

Morning

### --feedback--

The phrase "buenas tardes" is not used in the morning. Try again!

---

Evening

### --feedback--

> This greeting is used earlier in the day, not in the evening.

Author: ```suggestion
The greeting Camila says used earlier in the day, not in the evening.
```
